applications/machine_learning/README.rst

```diff
@@ -196,6 +220,11 @@ By default, the following buttons are used by the application:
   * **Button 1** switches between data forwarding and running a machine learning model.
   * **Button 3** changes the signal generated by the simulated sensor.
 
+* nRF5340 Development Kit:
+
+  * **Button 1** switches between data forwarding and running a machine learning model.
+  * **Button 3** changes the signal generated by the simulated sensor.
+
```

Comment:
Couldn't this be squashed to previous bullet point? Like:
`nRF52840 and nRF5340 Development Kit:`

Reply:
This is also defined separately for Thingy boards